run.py

```diff
@@ -71,6 +71,7 @@
         [--custom-config-file <file>]
         [--xunit-results]
         [--enable-eus]
+        [--skip-subscription-repos]
```

Comment:
`--skip-subscription-repos` actually sounds incorrect. we are not skipping subscription here, we just not enabling rhel rpms. can you use a different verbiage here like `skip-enabling-rhel-rpms` or something similar.

Reply:
Agree. Made the suggested changes.